### DISCUSSION TITLE
Can I add myst-parser extensions to mystmd documents?

### DISCUSSION THREAD

**Discussion by paulskrzypczyk** [executablebooks/meta#1240] [Q&A]
Apologies if this is a silly/obvious question. I've tried to answer from the website, but couldn't see if I'm missing something. 

I am interested in using, if possible, some of the extensions listed here: https://myst-parser.readthedocs.io/en/latest/syntax/optional.html inside a set of lecture notes I'm preparing using mystmd. For example, the typography extension would be very useful, as would block-attributes, for changing the numbering on lists beyond simple numeric.

As far as I could tell, this isn't possible, but I wanted to double check. If not, is this something that could potentially be implemented in the future?

Many thanks for the help. 

Paul

**Comment by agoose77** [comment]
@paulskrzypczyk thank you for your question! 

Your conclusion is correct that syntax extensions in `mystmd` are not yet configurable. We have `smartquotes` enabled, but not he replacements.

Block attributes are not a part of `mystmd`, yet. Those kind of syntax changes will go through an MEP process: https://mep.mystmd.org -- we would like to address the need for block markup.
